ptx/sec_limit_analytically.ptx

```diff
@@ -665,10 +708,21 @@
               \end{tikzpicture}
             </latex-image>
             <description>
-
+              <p>
+                Sector of a circle from <xref ref="img_squeeze1a"/> with dotted lines
+                creating the Right triangle from <xref ref="img_squeeze1a"/>, there is 
+                also a dotted line showing the same Acute triangle from 
+                <xref ref="img_squeeze1a"/>. The Acute triangle is contained inside
+                the circle sector, while the Right triangle purtrudes out of it.
+              </p> 
+              <p>
+                There is an angle <m>\theta</m> that is shared by both triangles and 
+                the circle sector. They also share the length 1 for the horizontal line.
+              </p>
             </description>
             <shortdescription>
-              
+              Sector of a circle from Figure 1.3.15a with dotted lines
```

Comment:
Best to avoid hard-coded figure numbers, because they could change if any content is added/removed. Something like "the previous image" would be better.